### ISSUE TITLE
Dynamically updating route with GeoJSON

### ISSUE THREAD

**Issue by shaunmulligan** [ngyewch/svelte-leaflet#24]
Hi, thanks for the great module! I am currently trying to create a map with a live updating route/track of where my device has been. Is there a way to do this currently? Also is there a way to bind a geojson object to the GeoJson component rather than giving it a url?

Alternatively is there a way to force the GeoJson component to re-render every couple of seconds to fetch the new data from the url?

**Comment by shaunmulligan** [comment]
I've currently made it work using `{#key value}` and an interval, but it feels a bit hacky...

```
let updateTrack = true
function routeReloadTimer() {
            let timer = setInterval(function() {
                updateTrack = !updateTrack
            }, 1000);
        }
routeReloadTimer()
</script>

<div class="map" >
<LeafletMap bind:this={leafletMap} options={mapOptions = {center: [$myStore.latitude, $myStore.longitude], zoom: 17}} >
    <TileLayer url={tileUrl} options={tileLayerOptions} />
    <Marker latLng={[$myStore.latitude, $myStore.longitude]}/>
    
    {#key updateTrack} 
        <GeoJSON url="/track.geojson" options={geoJsonOptions}/>
    {/key}
</LeafletMap>
</div>
```

**Comment by rgieseke** [comment]
Would the approach proposed in https://github.com/ngyewch/svelte-leaflet/pull/23 work for your usecase?

**Comment by shaunmulligan** [comment]
Yes, I believe #23 will do what I need! Will that be merged or should I use the fork?

**Comment by rgieseke** [comment]
Maybe testing it and reporting back on the PR might help. I haven't fully tested this (as i don't know all potential use cases) so would be great if you could test it!

**Comment by shaunmulligan** [comment]
No problem,  I will give it a test as soon as I can 👍

**Comment by rgieseke** [comment]
Up to @ngyewch to decide of course, i just meant it might be helpful to have more than person having played with the PR. :slightly_smiling_face:

**Comment by shaunmulligan** [comment]
ah right, I was confused about who I was chatting to :P

**Comment by ngyewch** [comment]
Done, I just merged #23. I hope this suits your needs.

**Comment by shaunmulligan** [comment]
thanks @ngyewch and @rgieseke this is working well now for my use case, so I will close the issue
